.github/workflows/phar.yaml

```diff
@@ -0,0 +1,53 @@
+name: PHAR build
+
+on:
```

Comment:
What's the recommended strategy for Phar build ? Should it be on every commit/pull request or only on release ?

Reply:
on release it's enough.

Reply:
Then can you update the pr in order to trigger it only on release (and maybe we keep manual dispatch) ? (Or did I misunderstand the current config)

Reply:
I thought about

´´´
on:
    release:
        types: [ published ]
    workflow_dispatch:
´´´

It will automatically run the github actions after I tag a release (or should the phar be generated BEFORE the release ?)

Reply:
I cannot help you on this purpose, because personnally, I prefer to use another approach with https://github.com/softprops/action-gh-release combined with https://github.com/miniscruff/changie/ to generate CHANGELOG full and chunk of release version.

For example: https://github.com/llaville/sarif-php-sdk/blob/master/.github/workflows/release.yml

Reply:
The box project seems to be have a github action to upload the phar on release created https://github.com/box-project/box/blob/main/.github/workflows/release.yaml#L83

I'll take a look